examples/nodejs/mongodb-examples/simple-read-aside/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": {
```

Comment:
i would expect to see a `scripts` section in here.

it might not be absolutely necessary but it usually helps developers figure out what options they have for how to build/consume the project.

Can you amend your README to include the npm/npx commands you are expecting users to run?  then i can pull this down and try it out and let you know whether i think the `scripts` section would be valuable.